gatsby-node.js

```diff
@@ -164,13 +170,6 @@ const processData = async (row, { createNodeId, createNode, store, cache, defaul
     }
   });
 
-  Object.keys(defaultValues).forEach(key => {
-    const cleanedKey = cleanKey(key);
-    if (processData[cleanedKey] === undefined) {
-      processData[cleanedKey] = defaultValues[key];
-    }
-  })
-
   // where childNodes returns an array of objects
   return { data: processedData, childNodes: childNodes };
```

Comment:
I think it makes sense to spread here for now. We can catch any user error later in key entry when we land a PR for #32.
```
return { data: {...defaultValues, ...processedData}, childNodes: childNodes };
```